src/pages/SubmissionDetails.js

```diff
@@ -36,34 +39,80 @@ export const SubmissionDetails = ({ submissionId }) => {
       }
     }
 
+    // TODO: "Prework" for the next ticket
```

Comment:
is the idea that this will get un-commented in the next ticket?

If so: I get it, but would still prefer not to check in this sort of thing...

Reply:
It is, but I can get rid of it!